homeassistant/components/lastfm/sensor.py

```diff
@@ -35,95 +32,57 @@
 )
 
 
+def format_track(track: Track) -> str:
+    """Format the track."""
+    return f"{track.artist} - {track.title}"
+
+
 def setup_platform(
     hass: HomeAssistant,
     config: ConfigType,
     add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the Last.fm sensor platform."""
-    api_key = config[CONF_API_KEY]
-    users = config[CONF_USERS]
-
-    lastfm_api = lastfm.LastFMNetwork(api_key=api_key)
-
+    lastfm_api = LastFMNetwork(api_key=config[CONF_API_KEY])
     entities = []
-    for username in users:
+    for username in config[CONF_USERS]:
         try:
-            lastfm_api.get_user(username).get_image()
-            entities.append(LastfmSensor(username, lastfm_api))
-        except WSError as error:
-            _LOGGER.error(error)
-            return
-
+            user = lastfm_api.get_user(username)
+            entities.append(LastFmSensor(user, lastfm_api))
+        except WSError as exc:
+            LOGGER.error("Failed to load LastFM user `%s`: %r", username, exc)
```

Comment:
Please keep the old `return` for now also.
```suggestion
            LOGGER.error("Failed to load LastFM user `%s`: %r", username, exc)
            return
```

It's a change of behavior and can be a dedicated follow-up.